AK/windows.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2024, stasoid <stasoid@yahoo.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
```

Comment:
This header file is not self-contained. If we want this and intend to use it from other impl files, please make sure to include AK/Platform.h.